src/Lachain.Core/Blockchain/VM/ExternalHandler.cs

```diff
@@ -36,18 +38,18 @@ public class ExternalHandler : IExternalHandler
             context.Message = message;
             return ContractInvoker.Invoke(address, context, input, gasLimit);
         }
-        
```

Comment:
For the future: don't add formatting changes to the functional pull request,  it makes review much more difficult

Reply:
Sure. It was my bad. I did auto format.